README.md

```diff
@@ -2,55 +2,158 @@
+let canHumansFlyAndIsTheSkyBlue = canHumansFly && isTheSkyBlue // false, because canHumansFly is false
+let canHumansFlyAndDolphinsWalk = canHumansFly && canDolphinsWalk // false, because both values are false
 ```
 
-* Explain and use all the various comparison operators:
-	* Equal to (a == b)
-	* Not equal to (a != b)
-	* Greater than (a > b)
-	* Less than (a < b)
-	* Greater than or equal to (a >= b)
-	* Less than or equal to (a <= b)
-	
+Since the statement will be `false` if a single value is `false`, an expression `canHumansFlyAndIsTheSkyBlue` will stop evaluating after checking `canHumansFly`, because that is already `false`. This method is called *short circuiting*. If the second value is the result of a function, that function would not even be called:
+
+```swift
+func printIfCalled() -> Bool {
+	print("Function is called.")
+	return true
+}
+
+let fullEvaluation = true && printIfCalled() // Will print "Function is called."
+let shortCircuit = false && printIfCalled() // Nothing will be printed, because of short circuiting
+```
 
-* Explain and use all of the various logical operators:
-	* Logical NOT (!a)
-	* Logical AND (a && b)
-	* Logical OR (a || b)
+In contrast to that, the logical `OR` operator will evaluate to `true` if at least one of the values is `true`:
 
 ```swift
-1 == 1   // true because 1 is equal to 1
-2 != 1   // true because 2 is not equal to 1
-2 > 1    // true because 2 is greater than 1
-1 < 2    // true because 1 is less than 2
-1 >= 1   // true because 1 is greater than or equal to 1
-2 <= 1   // false because 2 is not less than or equal to 1
+let isTheSkyBlue = true
+let isTheGrassGreen = true
+let canHumansFly = false
+let canDolphinsWalk = false
+
+let isTheSkyBlueOrTheGrassGreen = isTheSkyBlue || isTheGrassGreen // true, because both values are true
+let canHumansFlyOrIsTheSkyBlue = canHumansFly || isTheSkyBlue // true, because isTheSkyBlue is true
+let canHumansFlyOrDolphinsWalk = canHumansFly || canDolphinsWalk // false, because both values are false
 ```
 
+We already saw the logical `OR` in action, without having to write it out when we used the *less than or equal* operator earlier. We can write a function that works exactly like that operator, using just the *less than* and the *equal to* operator:
 
-## What the student can do at this point 
+```swift
+func lessThanOrEqual(first: Int, to second: Int) -> Bool {
+	return first < second || first == second
+}
 
-* Create variables and constants
-* Is familiar with type annotations, type inference and string interpolation.
-* Can create functions with return types.
-* Is familiar with the String, Int & Double type.
+let A = 7
+let B = 12
+let C = 12
 
+let isAlessThanOrEqualToB = lessThanOrEqual(A, to: B) // true, because 7 is less than 12
+let isBlessThanOrEqualToC = lessThanOrEqual(B, to: C) // true, because 12 is equal to 12
+let isClessThanOrEqualToA = lessThanOrEqual(C, to: A) // false, because 12 is not less than 7 and it is also not equal to 7
+```
```

Comment:
Similar to above, taking out the variables:

```swift
lessThanOrEqual(A, to: B) // true, because 7 is less than 12
lessThanOrEqual(B, to: C) // true, because 12 is equal to 12
lessThanOrEqual(C, to: A) // false, because 12 is not less than 7 and it is also not equal to 7
```